### PR TITLE
Added Teams Page

### LINKED ISSUE #5: Create Our Teams page

### PR DESCRIPTION
**Pull request by GenDelta**
Added the following: 
- Global component skew button
- Image SVGs of the icons of Linkedin, Github and Instagram for usage in the teams page
- TeamMembers.jsx and index.jsx for the Teams page in the src/pages/Teams folder

Rendered page looks like this:
image

Closes #5 
Kindly review the changes.

### CODE REVIEW

**Comment by Miran-Firdausi** [comment]
Why am i seeing no changes in the PR?

Did you committed and pushed all the changed files?

File: src/pages/Teams/TeamMember.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <p className="font-sans text-center text-wrap text-white">
+          {description}
+        </p>
+        <div className="flex justify-center space-x-4">
+          <a
+            href={linkedin}
+            target="_blank"
+            rel="noreferrer"
+            className="text-white font-sans"
+          >
```
Comment (Achyut-0705) [inline src/pages/Teams/TeamMember.jsx]: use Link from react-router-dom lib

File: src/pages/Teams/index.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    image:
+      "https://media.licdn.com/dms/image/C4D03AQFf0M0m5SBEsw/profile-displayphoto-shrink_800_800/0/1661514280105?e=1714608000&v=beta&t=OU8Rye-qRguvPyAsyJ19wMr3toD5PtIFWEwjORVlohc",
+    description:
+      "Praesent vestibulum libero non diam Praesent vestibulum libero non diam ",
+    linkedin: "https://linkedin.com/in/john-doe",
+    instagram: "https://instagram.com/john_doe",
+    github: "https://github.com/john-doe",
+  },
+  // Your team members data here
+];
```
Comment (Achyut-0705) [inline src/pages/Teams/index.jsx]: Export this to json and then use

**Review by Achyut-0705 — COMMENTED** [review]
Implement these changes

**Review by Achyut-0705 — APPROVED** [review]
LGTM! Good code quality, liked it

**Pull request merged**